network-api/networkapi/settings.py

```diff
@@ -628,7 +628,7 @@ class DatabasesDict(TypedDict):
 # Security
 SECURE_BROWSER_XSS_FILTER = env("XSS_PROTECTION")
 SECURE_CONTENT_TYPE_NOSNIFF = env("CONTENT_TYPE_NO_SNIFF")
-SECURE_CROSS_ORIGIN_OPENER_POLICY = env("SECURE_CROSS_ORIGIN_OPENER_POLICY", default="'same-origin'")
+SECURE_CROSS_ORIGIN_OPENER_POLICY = env("SECURE_CROSS_ORIGIN_OPENER_POLICY", default="same-origin-allow-popups")
```

Comment:
Let's keep the default to the safest option (i.e., `same-origin`), similar to how we are setting `self` as the default value for the CSP.

Reply:
Thanks for pointing that out how we're handling the defaults @mmmavis !